addons/mail/models/mail_channel.py

```diff
@@ -847,8 +851,14 @@ def channel_info(self, extra_info=False):
                 # avoid sending potentially a lot of members for big channels
                 # exclude chat and other small channels from this optimization because they are
                 # assumed to be smaller and it's important to know the member list for them
-                partner_ids = channel_partners.mapped('partner_id').ids
-                info['members'] = [partner_infos[partner] for partner in partner_ids]
+                members = channel_partners.read(['id', 'partner_id'])
```

Comment:
Make a channel member format method, as we will quickly need to add more information on it.